atomicapp/utils.py

```diff
@@ -352,6 +354,31 @@ def getRoot():
         else:
             return "/"
 
+    @staticmethod
+    def getLockFolder():
+        if os.access(constants.LOCK_FOLDER, os.W_OK):
```

Comment:
I'm confused. Where is the code to differentiate between a test and an actual `atomicapp run`?

From your PR notes it says that during a test it uses the .tmp dir while normally it'll use /run/lock